README.rst

```diff
@@ -12,13 +12,31 @@ Sequence Independent Molecular Replacement Based on Available Database
 
 .. image:: https://img.shields.io/badge/solution%20count-4-blue.svg?style=flat
    :target: http://www.rcsb.org/pdb/results/results.do?tabtoshow=Current&qrid=C03A7140
-   :alt: Solution count 
+   :alt: Solution count
 
-Documentation & Usage
-+++++++++++++++++++++
-* Please refer to `SIMBAD's installation page <http://simbad.readthedocs.io/en/latest/install.html>`_ for information on how to install SIMBAD and to maintain SIMBAD databases.
-* Please refer to `SIMBAD's examples page <http://simbad.readthedocs.io/en/latest/examples.html>`_ to see examples of how to run SIMBAD.
-* Please refer to `SIMBAD's documentation page <http://simbad.readthedocs.io/en/latest/contents.html>`_ for additional command line options and for API documentation.
+About
++++++
+
+SIMBAD is a sequence independant molecular replacement pipeline developed by the Rigden Lab at the University of Liverpool.
+SIMBAD provides an alternate strategy to identify search models for molecular replacement without using sequence information.
+This makes it suited to solve cases of contaminant crystallisation, and other mishaps such as mistaken identity (swapped crystallisation trays),
+as well as solving unsequenced targets and providing a brute-force approach where sequence-dependent search model identification
+could be non-trivial e.g. because of conformational diversity among identifiable homologues.
+
+Flowchart
++++++++++
+
+SIMBAD implements a three-step pipeline to efficiently identify a suitable search model in a database of known structures.
+
+1. SIMBAD performs a lattice search against the entire `Protein Data Bank <https://www.rcsb.org/>`_, rapidly determining whether or not a homologue exists in the same crystal form.
+2. SIMBAD screens the target data for the presence of a crystallised contaminant, a not uncommon occurrence in macromolecular crystallography. To catch for this eventuality, SIMBAD rapidly screens the data against a database of known contaminant structures.
```

Comment:
"a database of known contaminant structures. This database is compiled to include entries from ContaMiner, etc ..." --- add links as well. People would otherwise assume your contaminants might be different to ContaMiner and others